.github/README.md

```diff
@@ -0,0 +1,182 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h2><b>Leaderboard</b></h2>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Leaderdoard\] ](#-leaderdoard-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderdoard] <a name="about-project"></a>
+
+**[Leaderboard]** Is a website project built with webpack files that allows users to load scores.
+
+## 🛠 Built With <a name="built-with"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp">Css</a></li>
+    <li><a href="https://www.w3schools.com/javascript/default.asp">JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML Form]**
+- **[css flexbox]**
+- **[Semantic tags]**
+- **[Javascript]**
+- **[ES6 modules]**
+- **[webpack]**
+- **[Async]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps downbelow!
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web browser
+- Git-syn
+- Code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/Bleu-Yves-Sopoude/Leaderboard.git
+
+### Install
+
+Install this project run the command:
+
+```
+
+npm install
+
+```
+
+### Usage
+
+To run the project:
+
+- npm run build
+- npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+- Track HTML linter errors run:
+
+```
+npx hint .
+```
+
+- Track CSS linter errors run:
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+- Track JavaScript linter errors run:
+
+```
+npx eslint .
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Bleu Yves Sopoude (MrBlue)**
+
+- GitHub: [@Bleu-Yves-Sopoude](https://github.com/Bleu-Yves-Sopoude)
+- Linkedin: [@in/bleu-yves](https://www.linkedin.com/in/bleu-yves/)
+- GitHub: [@bleuYves](https://twitter.com/bleuYves)
+
```

Comment:
- [ ] [OPTIONAL] I believe you mistyped the Twitter handle for GitHub here as it redirects to your Twitter account. ✔